app/views/teacher_training_adviser/steps/_overseas_callback.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= f.govuk_fieldset legend: { text: 'Choose a time' } do %>
+
+<div class="govuk-form-group">
+  <%= f.label :phone_call_scheduled_at, "Select your preferred day and time for a callback", class: "govuk-label" %>
+  <%= f.select :phone_call_scheduled_at, callback_options(f.object.class.callback_booking_quotas), {}, { class: "govuk-select" } %>
```

Comment:
Doesn't matter too much here, but the latest version of the govuk form builder has a [new `#govuk_select` method](https://govuk-form-builder.netlify.app/form-elements/select/#taking-greater-control-over-the-options) which is a direct wrapper for `#select`

Reply:
Oh nice! I'll update that - I tried using the `govuk_collection_select` but it was going to be more work to manually churn out the options in the correct format